waypoints_x: [ 0.0, 20.0, 50.0, 80.0, 90.0, 100.0, 120.0 ]
waypoints_y: [ 0.0, 20.0, 50.0, 80.0, 90.0, 100.0, 120.0 ]
